src/pushsource/_impl/model/ami.py

```diff
@@ -71,3 +71,39 @@ class AmiPushItem(VMIPushItem):
         validator=optional(instance_of(AmiBillingCodes)),
     )
     """Billing codes associated with this image."""
+
+    image_name = attr.ib(type=str, default=None, validator=optional_str)
+    """image name  """
+
+    description = attr.ib(type=str, default=None, validator=optional_str)
+    """A description for the image"""
+
+    arch = attr.ib(type=str, default=None, validator=optional_str)
+    """Architecture type of the image"""
```

Comment:
This one we can get from `AmiRelease(VMIRelease)` as defined [here](https://github.com/release-engineering/pushsource/blob/master/src/pushsource/_impl/model/vms.py#L19-L20) thus it's not necessary here IMO

Reply:
Yeah that'd make sense.